bundle/bamoe/manifests/bamoe-kogito-operator.clusterserviceversion.yaml

```diff
@@ -988,7 +989,7 @@ spec:
   - automation
   labels:
     alm-owner-kogito: bamoe-kogito-operator
-    operated-by: bamoe-kogito-operator.8.0.3-2
```

Comment:
I think this is the same case we had for RHPAM Kogito, we updated to 8.0.3-2 but we did not release the Kogito bundle respin. So I think that the replaces should be indeed `replaces: bamoe-kogito-operator.8.0.3-1`